journal/modernized/08-chapter-7.adoc

```diff
@@ -240,14 +244,20 @@ General Meeting for Scotland, held at Edinburgh, 4th of Fifth Month, 1840.
 "`The foregoing Testimony on behalf of our late dear friend John Wigham,
 has been received from Aberdeen Monthly Meeting, and being approved,
 is signed on behalf of the meeting.`"
+
+[.signed-section-signature]
```

Comment:
I'm really not sure if any of these signatures are right in this Testimony chapter. Some of them seem to have long closings and nothing felt right about what to do with them. So it's a mess. 😢